compiler_base/tools/README.md

```diff
@@ -0,0 +1,60 @@
+# [WIP] compiler_base_test
+
+## Summary
+
+`compiler_base_test` is a part of [`compiler_base_tools`] crate that defines the core logic for unit tests execution engine. It aims to provide reusable components and abstract some functionalities for future developments.
+
+The `compiler_base_test` crate consists of three main components: Task, Executor, and Reporter.
```

Comment:
`Task` and `Executor` would be better to exist in packages other than `compiler_base_test`.